sdk/compute/Azure.ResourceManager.Compute/tests/Scenario/GalleryImageVersionContainerTests.cs

```diff
@@ -0,0 +1,216 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Compute.Models;
+using Azure.ResourceManager.Compute.Tests.Helpers;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Compute.Tests
+{
+    public class GalleryImageVersionContainerTests : ComputeTestBase
+    {
+        private ResourceGroup _resourceGroup;
+        private Gallery _gallery;
+        private GalleryImage _galleryImage;
+        protected GenericResourceContainer _genericResourceContainer;
+        private VirtualMachine _vm;
+        public GalleryImageVersionContainerTests(bool isAsync)
+           : base(isAsync , RecordedTestMode.Record)
+        {
+        }
+        private async Task<GalleryImageVersionContainer> GetGalleryImageVersionContainerAsync()
+        {
+            _resourceGroup = await CreateResourceGroupAsync();
+            var galleryName = Recording.GenerateAssetName("testGallery_");
+            var galleryImageName = Recording.GenerateAssetName("testGalleryImage_");
+            var input = ResourceDataHelper.GetBasicGalleryData(DefaultLocation);
+            var identifier = ResourceDataHelper.GetGalleryImageIdentifier(
+                    Recording.GenerateAssetName("publisher"),
+                    Recording.GenerateAssetName("offer"),
+                    Recording.GenerateAssetName("sku"));
+            var input_2 = ResourceDataHelper.GetBasicGalleryImageData(DefaultLocation, OperatingSystemTypes.Linux, identifier);
+            var lro_Gallery = await _resourceGroup.GetGalleries().CreateOrUpdateAsync(galleryName, input);
+            _gallery = lro_Gallery.Value;
+            var lro_GalleryImage = await _gallery.GetGalleryImages().CreateOrUpdateAsync(galleryImageName, input_2);
```

Comment:
C# usually do not use underscore as word separators
```suggestion
            var input_2 = ResourceDataHelper.GetBasicGalleryImageData(DefaultLocation, OperatingSystemTypes.Linux, identifier);
            var lroGallery = await _resourceGroup.GetGalleries().CreateOrUpdateAsync(galleryName, input);
            _gallery = lroGallery.Value;
            var lroGalleryImage = await _gallery.GetGalleryImages().CreateOrUpdateAsync(galleryImageName, input_2);
```